From: Andy Gospodarek <gospo@broadcom.com>
Date: Thu, 26 Apr 2018 17:44:38 -0400
Subject: bnxt_en: Increase RING_IDLE minimum threshold to 50
Patch-mainline: v4.18-rc1
Git-commit: 05abe4ddf0010e15419f5a6758b5bf44b7790982
References: bsc#1086282 FATE#324873

This keeps the RING_IDLE flag set in hardware for higher coalesce
settings by default and improved latency.

Signed-off-by: Andy Gospodarek <gospo@broadcom.com>
Signed-off-by: Michael Chan <michael.chan@broadcom.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/broadcom/bnxt/bnxt.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/broadcom/bnxt/bnxt.c
+++ b/drivers/net/ethernet/broadcom/bnxt/bnxt.c
@@ -7635,7 +7635,7 @@ static void bnxt_init_dflt_coal(struct b
 	coal->coal_bufs = 30;
 	coal->coal_ticks_irq = 1;
 	coal->coal_bufs_irq = 2;
-	coal->idle_thresh = 25;
+	coal->idle_thresh = 50;
 	coal->bufs_per_record = 2;
 	coal->budget = 64;		/* NAPI budget */
 
